risc0/zkvm/src/host/server/exec/tests.rs

```diff
@@ -49,11 +50,8 @@ fn run_test(spec: MultiTestSpec) {
         .unwrap()
         .build()
         .unwrap();
-    let session = ExecutorImpl::from_elf(env, MULTI_TEST_ELF)
-        .unwrap()
-        .run()
-        .unwrap();
-    assert_eq!(session.exit_code, ExitCode::Halted(0));
```

Comment:
I think at this level of testing, I was hoping to use `ExecutorImpl` directly... I have to think more about how this will effect things but wasn't the intent of these tests thus far.

Reply:
I've reverted these changes in https://github.com/risc0/risc0/pull/1241/commits/742143e47cd42335f0c2e38d9b23fe5c772a60fb